src/App.js

```diff
@@ -2,7 +2,7 @@ import { Routes, Route } from 'react-router-dom';
 import Layout from './components/Layout';
 import Rockets from './routes/Rockets';
 import MyProfile from './routes/MyProfile';
-import NotFound from './routes/NotFound';
+import Missions from './routes/Missions';
 
```

Comment:
@Talha-Hanif5666 please restore the "NotFound" since it's a valid route that will pop when someone gets a 404